libraries/Microsoft.Bot.StreamingExtensions/PayloadStream.cs

```diff
@@ -0,0 +1,179 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.StreamingExtensions.Payloads;
+
+namespace Microsoft.Bot.StreamingExtensions
+{
+    internal class PayloadStream : Stream
+    {
```

Comment:
There is a namespace / folder called "Payloads". This stream should be in there.